etc/firebase-admin.firestore.api.md

```diff
@@ -100,11 +100,20 @@ export function getFirestore(): Firestore;
 // @public
 export function getFirestore(app: App): Firestore;
 
+// @beta
+export function getFirestore(databaseId: string): Firestore;
+
+// @beta
+export function getFirestore(app: App, databaseId: string): Firestore;
+
 export { GrpcStatus }
 
 // @public
 export function initializeFirestore(app: App, settings?: FirestoreSettings): Firestore;
 
+// @beta
+export function initializeFirestore(app: App, settings: FirestoreSettings, databaseId: string): Firestore;
+

```

Comment:
Would it make more sense to include the `databaseId` in `FirestoreSettings`?

Reply:
TLDR; No, I I think this is a bad idea.

The admin SDK manages lifecycle of Firestore instances, such that subsequent requests will return the same instance. The `app` and `databaseId` form the key to this cache, and should really be held separate from settings in my opinion.

Other methods such as `getFirestore`, do not take a settings object. Adding a settings object here would conflict with intent to have `getFirestore` simply be a method to return an existing instance (if one exists).

`initializeFirestore` cannot be called with different settings, since settings cannot be changed after Firestore instance has been started. The semantic difference that "some" settings such as `database id` are allowed to change, will simply add confusion to interface. A simple rule that no settings are allowed to change can be maintained by excluding databaseId from settings.